aten/src/ATen/native/mps/operations/TensorCompare.mm

```diff
@@ -268,6 +269,52 @@ static void clamp_scalar_out_mps(const Tensor& input_t,
   }
 }
 
+static void isin_Tensor_Tensor_out_mps(const Tensor& elements,
+                                       const Tensor& test_elements,
+                                       bool assume_unique,
+                                       bool invert,
+                                       const Tensor& out,
+                                       string op_name) {
+  if (elements.numel() == 0) {
```

Comment:
Don't you want to check if `test_elements` is empty than you return True?

Reply:
Yeah good point, the implementation was not handling empty `test_elements` correctly. Included this as well as empty `elements` in the test cases and added the correct handling.